lib/types/index.d.ts

```diff
@@ -1396,6 +1396,11 @@ export class ESLint {
 
     static readonly version: string;
 
+    /**
+     * The default configuration that ESLint uses internally. This can change between versions, so it's not recommended to rely on this object.
```

Comment:
```suggestion
     * The default configuration that ESLint uses internally. This can change between versions, so relying on this object is not recommended.
```